sdks/wasm/Makefile

```diff
@@ -665,7 +665,7 @@ clean: clean-browser-tests
 	$(RM) -r *.o
 	$(RM) -r ./obj
 	$(RM) -r ./bin
-	$(RM) -r .stamp-build-debug-sample
```

Comment:
Should this line be kept for those who update master w/o running `make clean`? Won't there be a floating `.stamp-build-debug-sample` file that will be unused/is that okay?